activities/communication/blogging.md

```diff
@@ -0,0 +1,98 @@
+---
+Type: guide
+Explains: how to blog
+---
+
+# How and why we blog
+
+[We blog](https://blog.publiccode.net/) for a number of different reasons. This guide gives background information about the blog and instructions on how to blog.
+
+## Why we have a blog
+
+The blog is our primary tool for talking about:
+
+* our work: what we’re working on and what we’ve learned
+* our news: what we’ve achieved and what we’ll be doing next
+* our community: interviews with and guest posts by community members
+* our position on important issues: why we’ve made the political, technological, organizational or design decisions we have
+
+Not all information published on the blog needs to support our marketing strategy - for example, an in-depth discussion about a technology choice we’ve made may not persuade potential Foundation members, but will be interesting to technologists in our community.
+
+To make sure we have the editorial freedom to publish about all of our work (no matter how niche), there’s no direct link to the blog on [publiccode.net](https://publiccode.net/). Instead, we may feature important blogposts. (We’ll build this functionality once we have important blogposts.)
+
+## Blogposts are ephemera
+
+Blogposts are different from [resources and guides published on About](../documentation/index.md), like a white paper we keep updated. By contrast, a blogpost is only true at that moment in time. We shouldn’t ever update a blogpost with more current information.
```

Comment:
The first sentence is a bit ambiguous. Is it the blogposts that are like white papers? Perhaps add a "who are" after the comma and before "like"?

Reply:
I've clarified this now.